editor/src/core/model/scene-id-utils.ts

```diff
@@ -11,10 +11,10 @@ import * as PP from '../shared/property-path'
 import { assertNever } from '../shared/utils'
 import { isSceneAgainstImports, isRemixSceneAgainstImports } from './project-file-utils'
 
-const IdPropName = 'id'
+export const SceneCommentIdPropName = 'data-comment-id'
```

Comment:
Since it's ONLY for scenes, could we make this just `commentId`? Scene _is_ our component, so we we can make it a bit less clunky

Reply:
sure, I updated it in https://github.com/concrete-utopia/utopia/pull/4795/commits/7ebe371b3241a3cb295dc4d1d53d79b422cbbee3.

The reason I went with `data-comment-id` is because it fit the mold of the other Utopia-specific props on Scene (`data-label`, `data-uid`), and using `data-comment-id` didn't make it necessary to tweak the code of `Scene` (without the tweaking, a React error is triggered: `React does not recognize the 'commentId' prop on a DOM element`, since all props on Scene are forwarded to the underlying `div`).